src/trans/internal/trgtol_mod.F90

```diff
@@ -472,7 +472,7 @@ SUBROUTINE TRGTOL_COMM(PGLAT,KF_FS,KF_GP,KF_SCALARS_G,KVSET,&
 INTEGER(KIND=JPIM) :: ISEND, ITAG, JBLK, JFLD, JK, JL, IFLD, II, IFLDS, INS, INR
 INTEGER(KIND=JPIM) :: JJ,JI,IFLDT, J
 
-INTEGER(KIND=JPIA) :: JFLD64
+INTEGER(KIND=JPIB) :: JFLD64
```

Comment:
This variable was introduced in my CY49R1 [sync commit](https://github.com/ecmwf-ifs/ectrans/commit/635f9bb1c36c40945077e8f7e1ca68649ad794e6#diff-d50752e10eeb0069e066a6ac0bd9fea3ed72921d7b40aa07cfb5edca47d6efd2R475). I think @RyadElKhatibMF picked `JPIA` for a reason but not sure why. Can we keep it?